pkg/apis/serving/v1/configuration_validation.go

```diff
@@ -65,16 +64,8 @@ func (cs *ConfigurationSpec) Validate(ctx context.Context) *apis.FieldError {
 func (c *Configuration) validateLabels() (errs *apis.FieldError) {
 	for key, val := range c.GetLabels() {
 		switch key {
-		case serving.RouteLabelKey,
-			serving.ConfigurationUIDLabelKey,
-			serving.ServiceUIDLabelKey:
-			// Known valid labels - so just skip them
 		case serving.ServiceLabelKey:
```

Comment:
With only one 'case' should we just drop the switch and directly access this one key in the label map?

Reply:
sure